deployment/src/kubernetes/README.md

```diff
@@ -165,7 +165,7 @@ $ doctl compute firewall get <ID> --context <context-name>
 
 ***ATTENTION:** This seems not to work at all so we leave it away at the moment*
 
-***TODO:** I thought this is necessary if we use the DigitalOcean DNS management service? See [Manage DNS With DigitalOcean](/deployment/kubernetes/DigitalOcean.md#manage-dns-with-digitalocean)*
+***TODO:** I thought this is necessary if we use the DigitalOcean DNS management service? See [Manage DNS With DigitalOcean](./DigitalOcean.md#manage-dns-with-digitalocean)*
```

Comment:
Absolut path?

Reply:
In case linked markdown files are in the same directory, relative paths work for both Github and Vuepress.